### ISSUE TITLE
interactive execution

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#932]
Allow TerraME to have a way to have interaction with the user, with execution option -interactive. This option should show a prompt for the user to describe commands and show their results before showing the prompt again. This will be useful mainly for packages that handle geospatial data, for example fillcell.

**Comment by tiagogsc** [comment]
This might be somehow similar to "terrame -GUI" which exhibits a Player GUI
that allows step-by-step  execution?

Em Seg, 29 de fev de 2016 10:40, Pedro R. Andrade <notifications@github.com>
escreveu:

**Comment by pedro-andrade-inpe** [comment]
No, it is like running Lua or R as a stand-alone application. You can write commands and see the result interactively. The session does not end after finishing the script and closing all observers.

**Comment by avancinirodrigo** [comment]
Is it like a Qt-Lua console?

**Comment by pedro-andrade-inpe** [comment]
Yes.

**Comment by tiagogsc** [comment]
In this case, the code is ready to use in Visual TerraME. Please get in
touch with Washington .

Em Seg, 29 de fev de 2016 14:57, Pedro R. Andrade <notifications@github.com>
escreveu:

**Comment by pedro-andrade-inpe** [comment]
Sorry, the "Yes" was related to Rodrigo's question. In fact, the answer to Tiago's question is no (see above).
